sections/model_overview/main.tex

```diff
@@ -1 +1,168 @@
+of individuals. 
+These are the individuals that the distributor allocates to them and 
+other individuals from other sources. 
+Each queueing system may then choose to block the individuals that arrive from 
+the distributor when the system reaches a certain capacity. 
+Thus, the set of strategies that they may choose to play in the game are defined 
+as the set of possible choices that the two queueing systems have. 
+In other words, each queueing system is given \( N_A \) and \( N_B \) number of 
+strategies where \( N_A \) and \( N_B \) are the total capacities of the two 
+queueing systems.
+The \textit{threshold} parameter is thus used to capture the chosen 
+strategy for each hospital where \( T_A \in \left[1, N_A \right] \) and 
+\( T_B \in \left[1, N_B \right] \)
+Their choice of strategy will then rely solely on satisfying their own 
+objective, which is to make sure that the waiting time of a proportion of 
+individuals will be below the predefined target time.
+
+\begin{equation}
+    P(W < \textit{ Target}) \geq \hat{P}
+\end{equation}
+
+where \(W\) is the waiting time of all individuals, \textit{Target} is the time 
+target and \(\hat{P}\) is the percentage of individuals need to be within that 
+target. 
+There are numerous objective functions that can be used to capture this 
+behaviour. 
+The first one would be to use the threshold that maximises the probability that 
+the mean waiting is more than the target time, and completely ignore the 
+percentage goal.
+
+\begin{equation}
+    \arg \max_{T_i} \quad P(W_i < \textit{Target})
+\end{equation}
+
+Perhaps a more sophisticated objective function would be to get the proportion 
+of individuals as close to the percentage aim. 
+In other words, to find the threshold that minimises the difference between the 
+probability and the percentage goal (or maximise its negation).
+
+\begin{equation}\label{eq:obj-queueing-systems}
+    \arg \max_{T_i} \quad -\left( \hat{P} - P(W_i < \textit{Target}) \right)^2
+\end{equation}
+
+
+On the other side of the spectrum, the distributor has its own choices 
+to make and its own goals to satisfy.
+The set of strategies it has can be defined as the different ways that it can 
+distribute individuals to the two queueing system.
+Therefore, unlike the queueing systems, the distributor has a 
+continuous set of strategies that goes from \( 0\% - 100\% \). 
+For example, it may choose to send \(35\%\) of its individuals to the first 
+queueing system and \(65\%\) to the second one.
+In addition, the distributor aims to minimise any potential blockages
+that may occur, given the pair of thresholds chosen by the two queueing systems.
+Thus, its objective is to minimise the blocked time of the individuals 
+that they send to the two queueing systems.
+Apart from the time being blocked, an additional aspect that may affect the 
+decision of the distribution service is the proportion of lost individuals.
```

Comment:
We need to clarify above (somewhere near the diagram of the queueing system that individuals can be lost at all).